hammer/reporting-remediation/reporting/create_s3bucket_policy_issue_tickets.py

```diff
@@ -45,7 +45,26 @@ def create_tickets_s3buckets(self):
                     bu = issue.jira_details.business_unit
                     product = issue.jira_details.product
 
-                    if issue.status in [IssueStatus.Resolved, IssueStatus.Whitelisted]:
+                    if issue.status in [IssueStatus.Tempwhitelist] and issue.timestamps.temp_whitelisted is None:
+                        logging.debug(f"S3 bucket public policy issue '{bucket_name}' "
+                                      f"is added to temporary whitelist items.")
+
+                        comment = (f"S3 bucket public policy '{bucket_name}' issue "
+                                   f"in '{account_name} / {account_id}' account is added to temporary whitelist items.")
+                        jira.update_issue(
+                            ticket_id=issue.jira_details.ticket,
+                            comment=comment
+                        )
+
+                        slack.report_issue(
+                            msg=f"{comment}"
+                                f"{' (' + jira.ticket_url(issue.jira_details.ticket) + ')' if issue.jira_details.ticket else ''}",
+                            owner=owner,
+                            account_id=account_id,
+                            bu=bu, product=product,
+                        )
+                        IssueOperations.set_status_temp_whitelisted(ddb_table, issue)
+                    elif issue.status in [IssueStatus.Resolved, IssueStatus.Whitelisted]:
```

Comment:
Create a new ticket without comment, that ticket is temporary whitelisted.
Description contains "Auto-Remediation Date: 2019-10-18"
https://issues.staging.dowjones.net/browse/SECVULN-6352

Reply:
Agreed and will update changes.